### ISSUE TITLE
Create / Deploy Tiger BOM

### ISSUE THREAD

**Issue by svbergmann** [gematik/app-Tiger#14]
It would be very nice to have a tiger BOM for having all version of every plugin sorted out. Two examples are already given in the tiger user manual which are the [Cucumber-BOM](https://mvnrepository.com/artifact/io.cucumber/cucumber-bom) and the [JUnit-BOM](https://mvnrepository.com/artifact/org.junit/junit-bom).

**Comment by Arkinator** [comment]
Yes! Great idea. It was discussed internally, but since the user-need did not materialize the ticket floated quite far down in the backlog. Thanks for the initiative, we will revive it!

**Comment by eitzenbe** [comment]
Hi @svbergmann,
Just wondering for what exactly you would need the BOM and if you are referring to a maven starter pom like BOM or a real SBOM created by tools like Syft or sonartype, ....
Please share some more user point of view so that we can provide you with an appropriate solution ;)
br
Thomas

**Comment by svbergmann** [comment]
Hi @eitzenbe, 
I am referring to a BOM file where you put all versions of the dependencies so that the end-user does not have to worry too much about the versions for every plugin. 
Here is one example: 
We are using the plugins [tiger-rbel](https://mvnrepository.com/artifact/de.gematik.test/tiger-rbel), [tiger-proxy](https://mvnrepository.com/artifact/de.gematik.test/tiger-proxy), [tiger-test-lib](https://mvnrepository.com/artifact/de.gematik.test/tiger-test-lib), [tiger-testenv-mgr](https://mvnrepository.com/artifact/de.gematik.test/tiger-testenv-mgr), [tiger-common](https://mvnrepository.com/artifact/de.gematik.test/tiger-common), and [tiger-cloud-extension](https://mvnrepository.com/artifact/de.gematik/tiger-cloud-extension). All of them in the same version except from the last one. I think we would benefit from a BOM (like the above mentioned [Cucumber-BOM](https://mvnrepository.com/artifact/io.cucumber/cucumber-bom)) where all those versions are defined clearly so that we could have a slimmer pom as well as just need to update the BOM version instead of looking through every plugin to get the version.
I hope that helps! 
Kind regards
Sven

**Comment by eitzenbe** [comment]
Great thanx for the feedback we will look into this probably around october as we are quite busy with other stuff at the moment...
